pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[tool.isort]
+src_paths = [".", "tools"]
```

Comment:
Should be `["tools", "zulip", "zulip_bots", "zulip_botserver"]` since each of the latter three are their own roots with their own `setup.py`.